### PR TITLE
[azeventgrid(s)] Regenerating azsystemevents based on latest commit, updating CHANGELOG dates for release.

### PR DESCRIPTION
**Pull request by richardpark-msft**
- Regenerating azsystemevents based on latest approved commit.
- Adding release dates for the other packages for their first publish.

### CODE REVIEW

File: sdk/messaging/eventgrid/azsystemevents/models.go [hunk 1]
```diff
@@ -244,6 +270,15 @@ type APIManagementUserUpdatedEventData struct {
...
+type APISpecificationDetails struct {
+	// Specification name.
+	Name *string
+
+	// Specification version.
+	Version *string
+}
+
 // AcsChatEventBaseProperties - Schema of common properties of all chat events
 type AcsChatEventBaseProperties struct {
```
Comment (jhendrixMSFT) [inline sdk/messaging/eventgrid/azsystemevents/models.go:283]: Can we capitalize `Acs`?

File: sdk/messaging/eventgrid/azsystemevents/models.go [hunk 2]
```diff
@@ -1851,6 +1889,166 @@ type AppServicePlanEventTypeDetail struct {
 	Status *AsyncStatus
 }
 
+// AvsClusterCreatedEventData - Schema of the Data property of an CloudEvent/EventGridEvent for a Microsoft.AVS.ClusterCreated event.
+type AvsClusterCreatedEventData struct {
```
Comment (jhendrixMSFT) [inline sdk/messaging/eventgrid/azsystemevents/models.go:1893]: Capitalize `Avs`?

**Review by jhendrixMSFT — APPROVED** [review]
It would be ideal to capitalize the acronyms.

File: sdk/messaging/eventgrid/azsystemevents/models.go (hunk 2 above)
Comment (richardpark-msft) [reply sdk/messaging/eventgrid/azsystemevents/models.go:1893]: Ah, interesting. Yes, I can fix those. We'll have to get in the habit of reviewing these now, it's basically a monthly cadence at this point.

File: sdk/messaging/eventgrid/azsystemevents/models.go (hunk 1 above)
Comment (richardpark-msft) [reply sdk/messaging/eventgrid/azsystemevents/models.go]: Fixed! Also, got the bonus acronym of Iot capitalized.

File: sdk/messaging/eventgrid/azsystemevents/models.go (hunk 2 above)
Comment (richardpark-msft) [reply sdk/messaging/eventgrid/azsystemevents/models.go]: Fixed!

**Pull request merged**